pkg/install/upgrade.go

```diff
@@ -36,6 +36,7 @@ type UpgradeController struct {
 	privateLinkSecret         corev1.Secret
 	imageOverrideConfigmap    corev1.ConfigMap
 	reinstallNeeded           bool // this is used only for code test
+	awsPlatform               bool // this is used only for code test
```

Comment:
Is this really needed?

Reply:
I'm not sure how to expose the awsPlatform variable to the tests otherwise. If the tests aren't important I could remove it